### ISSUE TITLE
Accessibility issue in grid component

### ISSUE THREAD

**Issue by ExtAnimal** [bryntum/support#13069]
[Forum post](https://forum.bryntum.com/viewtopic.php?t=35301)

In out Bryntum instances we have two cases where both role="presentation" and aria-label="normal" are used. 

This can cause issues for:
Screen reader users, who may receive missing, confusing, or inconsistent
information about the grid structure

Keyboard-only users, if the semantic model of the widget does not match the
interactive experience

Users relying on assistive technology navigation, who may be unable to
understand regions, headers, or relationships within the planner/grid

The two places that we have observed this are: b-t-1-normalSubgrid-header 

Source:

<div
data-region="normal"
data-max-depth="0"
data-owner-cmp="b-t-1-normalSubgrid-header"
class="b-grid-headers b-grid-headers-normal b-widget-scroller b-resize-
monitored b-hide-scroll b-has-flex b-horizontal-overflow"
role="presentation"
style="overflow-x: auto;">
</div>

and 

b-t-1-normalSubgrid

Source:

<div
data-region="normal"
class="b-grid-subgrid ..."
aria-label="normal"
role="presentation"
id="b-t-1-normalSubgrid"
style="flex: 1 1 0%; ..."
data-ref="normal"
data-item-index="1">
</div>

I did look for examples of this in the Bryntum Demos and I could not find any for these specific elements. Maybe this is because it has already been fixed or maybe there is some difference in setup. 

Can you investigate this and let me know if the issue has been resolved or if it is going to be?

I hope this is enough information if you require anymore then please let me know?

Thank you very much,
Harry
